dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/DataRequestCompanyIdentifierType.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.enums.p2p
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum for the different identifier types that an identifier value of a stored data request can have.
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class DataRequestCompanyIdentifierType {
```

Comment:
Is there a test which makes sure that the different identifiers all work?

Reply:
@PaulChr99 has done heaps of testing. I guess he has tests vor all different cases here.

Reply:
Yes, the first backend e2e test covers all of those identifiers except for the company Id. For the latter, there are extra tests addressing the cases where either two identifiers in the same bulk request map to the same company (i.e. to the companyId) or one of the individual requests sent as part of the bulk request is already existing for that company (i.e. stored via its companyId as identifier)

Reply:
ok